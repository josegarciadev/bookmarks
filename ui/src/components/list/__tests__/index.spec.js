import { shallow } from 'enzyme';

import { BookmarkList } from '../index';

describe('BookmarkList', () => {
    let props={
       data:[
        {id:'48e6c5e7-e0c6-4614-8ac1-8193920de8eb',title:'google',link:'http://google.com'},
        {id:'92ac82a5-74ba-4a99-9ede-d9d2a434568f',title:'Bookmarks',link:'http://Bookmarks.com'},
        {id:'f1f20ffd-edf8-4f11-99ab-359158723eb1',title:'curso react native',link:'http://reactNative.com'},
        {id:'bbda2eb6-6917-47f0-a512-5903cc568c44',title:'curso flutter',link:'http://flutterCurso.com'},
       ]
    }
    it('should match snapshot', () => {
        const rendered = shallow(<BookmarkList {...props}/>);
        expect(rendered).toMatchSnapshot();
    });
});