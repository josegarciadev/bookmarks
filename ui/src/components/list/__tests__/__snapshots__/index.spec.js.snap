// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarkList should match snapshot 1`] = `
<div
  className="container"
>
  <div
    className="container-grid"
  >
    <Card
      key="48e6c5e7-e0c6-4614-8ac1-8193920de8eb"
      label="google"
    />
    <Card
      key="92ac82a5-74ba-4a99-9ede-d9d2a434568f"
      label="Bookmarks"
    />
    <Card
      key="f1f20ffd-edf8-4f11-99ab-359158723eb1"
      label="curso react native"
    />
    <Card
      key="bbda2eb6-6917-47f0-a512-5903cc568c44"
      label="curso flutter"
    />
  </div>
</div>
`;
